ivy/functional/frontends/jax/numpy/name_space_functions.py

```diff
@@ -224,6 +224,10 @@ def fmax(x1, x2):
     return ret
 
 
```

Comment:
Could you add the `@inputs_to_ivy_arrays` decorator here please?

Reply:
I added the other arguments into the return ivy function. Hope that works?